admin/tool/policy/lang/en/tool_policy.php

```diff
@@ -47,6 +52,7 @@
 $string['newpolicysite'] = 'Site policy';
 $string['newpolicythirdparties'] = 'Third parties policy';
 $string['newversion'] = 'New version';
+$string['explanationdigitalminor'] = 'Some explanation about digital minors and ensuring personal information is not stored / processed / etc.';
```

Comment:
If you leave unfinished strings like this, it may help to start it with something like `TODO` or so so that we can find all such places more easily.